ros_speech_recognition/README.md

```diff
@@ -30,6 +30,106 @@ This package uses Python package [SpeechRecognition](https://pypi.python.org/pyp
   print result # => 'Hello, world!'
   ```
   
+If you are using `ros_speech_recognition` with `~continuous` is `True`, you can subscribe `/Tablet/voice` (`speech_recognition_msgs/SpeechRecognitionCandidates`) message.
```

Comment:
> This is because the new fields (e.g. wordinfo) seems to be specific to the google cloud speech-to-text.

This is not specific to the google cloud speech-to-text.
For example, azure cognitive service has similar functions.
https://docs.microsoft.com/ja-jp/python/api/azure-cognitiveservices-speech/azure.cognitiveservices.speech.speechconfig?view=azure-python#request-word-level-timestamps--

I think that the start time and end time for each word are general values as a framework for speech recognition.

> This is my preference, but how about wrapping the common message by each speech-to-text service message?
> The advantage of this method is we do not need to change the current

However, this is a good way to avoid affecting other users, so I'll take this direction.